### DISCUSSION TITLE
Change Tracking Dev Repository from Re-Cap 2023

### DISCUSSION THREAD

**Discussion by tf484** [cap-js/.github#3] [General]
Hi,
I have watched the recording from re>=CAP 2023 Change Tracking with CAP, and I wanted to look at the implementation, but I could not find the repository Change Tracking Dev that in the video it is mentioned that can be found here, could I have access to the repository please?
Thanks,
Tatiana F.

**Comment by gregorwolf** [comment]
Hi @tf484,

Maybe it was just recently made public: https://github.com/cap-js/change-tracking
